core/server/master/src/main/java/alluxio/master/meta/checkconf/WrongProperty.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.meta.checkconf;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Records a property that is required or recommended to be consistent but is not within its scope.
+ *
+ * For example, the ConsistencyCheckLevel of key A is ENFORCE and its Scope is MASTER,
+ * so this property is required to be consistent in all master nodes.
+ * The ConsistencyCheckLevel of key B is WARN and its Scope is SERVER,
+ * so this property is recommended to be consistent in all master and worker nodes.
+ */
+public class WrongProperty {
```

Comment:
this is a confusing name. 
Maybe  `InconsistentProperty`?

Reply:
Renamed, thanks